Polyhedron/demo/Polyhedron/Plugins/Mesh_3/CMakeLists.txt

```diff
@@ -34,13 +34,13 @@ else()
   message(STATUS "NOTICE : the DICOM files (.dcm) need VTK libraries to be open and will not be able to.")
 endif()
 
-find_package(Boost QUIET OPTIONAL_COMPONENTS filesystem)
+find_package(Boost QUIET OPTIONAL_COMPONENTS filesystem system)
 if(Boost_FILESYSTEM_FOUND)
   qt5_wrap_ui( imgUI_FILES Image_res_dialog.ui raw_image.ui)
   polyhedron_demo_plugin(io_image_plugin Io_image_plugin Volume_plane_intersection.cpp Raw_image_dialog.cpp ${imgUI_FILES} ${VOLUME_MOC_OUTFILES} KEYWORDS IO Mesh_3)
   target_link_libraries(io_image_plugin PUBLIC scene_image_item ${VTK_LIBRARIES} CGAL::CGAL_ImageIO)
   if(TARGET Boost::filesystem)
-    target_link_libraries(io_image_plugin PUBLIC Boost::filesystem)
+    target_link_libraries(io_image_plugin PUBLIC Boost::filesystem Boost::system)
```

Comment:
I do not know why `Boost::system` is now required. Why is that in this PR?

If it is justified, then the `if(Boost_FILESYSTEM_FOUND)` should be updated to add the test for `system`, as well as the message when one of them is not found.